### DISCUSSION TITLE
MAIS GRUPOS HIERARQUICOS

### DISCUSSION THREAD

**Discussion by iEmersonJr** [RamonSilva20/mapos#2092] [General]
Saudações, pessoal. Conheci o Mapos e tenho gostado profundamente do sistema, finalizei a configuração dele e já está na hospedagem, porém, notei que só existe permissão de "Administrador" ou "Cliente", seria possivel criar mais permissões para um novo grupo a exemplo: Moderador (recepção, ajudante e afins...) Eu sou novato em programação, preciso de orientações para leigos. 

Vi que na base de dados, tem essas tabelas, mas como posso saber oque cada permissão significa?  
![image](https://user-images.githubusercontent.com/128086295/228662965-166d52db-31a1-48ac-bf19-7f9046986714.png)

as permissões de ADM: 

a:53:{s:8:"aCliente";s:1:"1";s:8:"eCliente";s:1:"1";s:8:"dCliente";s:1:"1";s:8:"vCliente";s:1:"1";s:8:"aProduto";s:1:"1";s:8:"eProduto";s:1:"1";s:8:"dProduto";s:1:"1";s:8:"vProduto";s:1:"1";s:8:"aServico";s:1:"1";s:8:"eServico";s:1:"1";s:8:"dServico";s:1:"1";s:8:"vServico";s:1:"1";s:3:"aOs";s:1:"1";s:3:"eOs";s:1:"1";s:3:"dOs";s:1:"1";s:3:"vOs";s:1:"1";s:6:"aVenda";s:1:"1";s:6:"eVenda";s:1:"1";s:6:"dVenda";s:1:"1";s:6:"vVenda";s:1:"1";s:9:"aGarantia";s:1:"1";s:9:"eGarantia";s:1:"1";s:9:"dGarantia";s:1:"1";s:9:"vGarantia";s:1:"1";s:8:"aArquivo";s:1:"1";s:8:"eArquivo";s:1:"1";s:8:"dArquivo";s:1:"1";s:8:"vArquivo";s:1:"1";s:10:"aPagamento";N;s:10:"ePagamento";N;s:10:"dPagamento";N;s:10:"vPagamento";N;s:11:"aLancamento";s:1:"1";s:11:"eLancamento";s:1:"1";s:11:"dLancamento";s:1:"1";s:11:"vLancamento";s:1:"1";s:8:"cUsuario";s:1:"1";s:9:"cEmitente";s:1:"1";s:10:"cPermissao";s:1:"1";s:7:"cBackup";s:1:"1";s:10:"cAuditoria";s:1:"1";s:6:"cEmail";s:1:"1";s:8:"cSistema";s:1:"1";s:8:"rCliente";s:1:"1";s:8:"rProduto";s:1:"1";s:8:"rServico";s:1:"1";s:3:"rOs";s:1:"1";s:6:"rVenda";s:1:"1";s:11:"rFinanceiro";s:1:"1";s:9:"aCobranca";s:1:"1";s:9:"eCobranca";s:1:"1";s:9:"dCobranca";s:1:"1";s:9:"vCobranca";s:1:"1";}

![image](https://user-images.githubusercontent.com/128086295/228663291-f5011250-5394-48d3-b259-2197b5e30bd3.png)

A de moderador como consta em anexo é copia do de ADM. Mas gostaria de limitar mais o acesso moderador.

**Comment by marlontomaz** [comment]
olá, Emerson... 
Pelo painel do sistema em configurações, click em permissoes , e inclua uma permissao: exemplo -> recepção ... e nessa nova permissao, informa o nivel de acesso.... tudo isso pelo painel do sistema, não precisa alterar o codigo.
